tests/acceptance/register-test.js

```diff
@@ -11,23 +11,24 @@ module('Acceptance | register', function(hooks) {
   setupLoggedOutUser(hooks);
 
   test('successful registration', async function(assert) {
-    const user = {
+    const user = this.server.create('user', {
```

Comment:
Use Mirage to prepopulate the db and add a `username` as that's what's being rendered in the nav